src/v/features/feature_table.h

```diff
@@ -99,6 +99,7 @@ inline const std::unordered_set<std::string_view> retired_features = {
   "mtls_authentication",
   "rm_stm_kafka_cache",
   "transaction_ga",
+  "idempotency_v2",
```

Comment:
can you make this a separate commit and remove the corresponding feature definition ?

Reply:
done.